packages/inquirer_oc/inquirer_oc.1.0.8/opam

```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A collection of common interactive command line user interfaces"
+description:
+  "A collection of common interactive command line user interfaces"
+maintainer: ["Ou Fumio"]
+authors: ["Ou Fumio"]
+license: "MIT"
+tags: [
+  "answer"
+  "answers"
+  "ask"
+  "base"
+  "cli"
+  "command"
+  "command-line"
+  "confirm"
+  "enquirer"
+  "generate"
+  "generator"
+  "hyper"
+  "input"
+  "inquire"
+  "inquirer"
+]
+homepage: "https://github.com/araera111/inquirer_oc"
+doc: "https://url/to/documentation"
```

Comment:
If you don't have a dedicated doc page, you can put a link to the dev repo again.